core/block/import/markdown/anymark/renderer.go

```diff
@@ -143,10 +131,10 @@ func (r *Renderer) renderFencedCodeBlock(_ util.BufWriter,
 		fields = &types.Struct{Fields: map[string]*types.Value{"lang": pbtypes.String(language)}}
 	}
 	if entering {
```

Comment:
133-137 - is that correct taking into account, that `openTextBlockWithStyle` contains if-else condition for `entering`?

Reply:
yes because we are also passing the field to the function